src/test/java/karate/balance/BalanceTest.java

```diff
@@ -23,6 +27,14 @@
 @DirtiesContext
 class BalanceTest extends BaseTest {
 
+    @BeforeEach
+    protected void setup() {
+        super.setup();
+
+        // Create test user for balance history tests
+        addUser("member2", TEST_USER_PASSWORD, 500f, List.of(Role.MEMBER));
```

Comment:
This user has a specific use for one test if I see that correctly? Please give it a more "documenting" name, that way we know in the future why this user exists

Reply:
Done, renamed `member2` to `balance_history_member` to indicate its purpose.